src/hooks/wallets/wallets.ts

```diff
@@ -14,10 +14,33 @@ import pairingModule from '@/services/pairing/module'
 import e2eWalletModule from '@/tests/e2e-wallet'
 import { CGW_NAMES, WALLET_KEYS } from './consts'
 
+// We need to modify the module name as onboard dedupes modules with the same label and the WC v1 and v2 modules have the same
+// @see https://github.com/blocknative/web3-onboard/blob/d399e0b76daf7b363d6a74b100b2c96ccb14536c/packages/core/src/store/actions.ts#L419
+const walletConnectV2 = (): WalletInit => {
+  return (helpers) => {
+    const MODULE_LABEL = 'WalletConnect v2'
```

Comment:
The plan was to disable v1 via the config-service, at which point this module should be called just `WalletConnect`. Can you use the name from the config-service instead of a hardcoded one?

Reply:
As discussed, due to the current limitations of wallet naming on the config service we will keep this as "WalletConnect" but alter the v1 module to be "WalletConnect v1" instead:

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/8b01742e-a5bc-4af2-9aa9-9e9b0c2bf39a)

We can/should cleanup the config service naming convention at some stage, for which we already have [an issue](https://github.com/safe-global/safe-wallet-web/issues/713).